content/docs/07-reference/ai-sdk-ui/01-use-chat.mdx

```diff
@@ -158,6 +158,82 @@ Allows you to easily create a conversational user interface for your chatbot app
       name: 'messages',
       type: 'Message[]',
       description: 'The current array of chat messages.',
+      properties: [
+        {
+          type: 'Message',
+          parameters: [
+            {
+              name: 'id',
+              type: 'string',
+              description: 'The unique identifier of the message.',
+            },
+            {
+              name: 'role',
+              type: "'system' | 'user' | 'assistant' | 'data'",
+              description: 'The role of the message.',
+            },
+            {
+              name: 'content',
+              type: 'string',
+              description: 'The content of the message.',
+            },
+            {
+              name: 'createdAt',
+              type: 'Date',
+              isOptional: true,
+              description: 'The creation date of the message.',
+            },
+            {
+              name: 'name',
+              type: 'string',
+              isOptional: true,
+              description: 'The name of the message.',
+            },
+            {
+              name: 'data',
+              type: 'JSONValue',
+              isOptional: true,
+              description: 'Additional data sent along with the message.',
+            },
+            {
+              name: 'annotations',
+              type: 'Array<JSONValue>',
+              isOptional: true,
+              description: 'Additional annotations sent along with the message.',
+            },
+            {
+              name: 'experimental_attachments',
+              type: 'Array<Attachment>',
+              isOptional: true,
+              description: 'Additional attachments sent along with the message.',
+              properties: [
+                {
+                  type: 'Attachment',
+                  parameters: [
+                    {
+                      name: 'name',
+                      type: 'string',
+                      isOptional: true,
+                      description: 'The name of the attachment.',
```

Comment:
would it make to clarify that this is a filename? maybe even renaming to filename?

Reply:
or is it more general?

Reply:
I'm leaning towards making it general since we chose to call it `attachments` instead of `files`. We also risk introducing two names describing the same thing.